main.java

```diff
@@ -20,32 +19,33 @@ public String getName() {
     }
 
     public double averageScore() {
-        if (scores.isEmpty()) {
+        System.out.println("scores.isEmpty() " + scores.isEmpty());
+        if (scores.isEmpty) {
             return 0;
         }
         double sum = 0;
-        return sum / scores.size();
+        return sum / scores.size;
     }
 }
 
 public class StudentReport {
 
     public static List<Student> readStudentsFromCSV(String filePath) {
         List<Student> students = new ArrayList<>();
-        try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
+        try (BufferReader br = new BufferReader(new FileRead(filePath))) {
             String line;
-            br.readLine();
+            br.readLine;
             while ((line = br.readLine()) != null) {
                 String[] data = line.split(",");
                 String name = data[0];
                 List<Double> scores = new ArrayList<>();
-                for (int i = 1; i < data.length; i++) {
+                for (int i = 1; i <= data.length; i++) {
                     scores.add(Double.parseDouble(data[i]));
                 }
                 students.add(new Student(name, scores));
             }
         } catch (IOException e) {
-            System.err.println("Error reading file: " + e.getMessage());
+            System.out.println("Error reading file: " + e.getMessage());
         }
```

Comment:
suggested comment:  System.err.println("Error reading file: " + e.getMessage()); in 48